LICENSE.md

```diff
@@ -16,23 +16,23 @@ Version 2.0.2
 
 **Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
 
-**API**: Software intermediary for interaction with the Mod.
+**API**: Software which is part of the Mod that allows other software to interact with the Mod.
 
-**Derivative**: Software which includes any portion of the Mod other than the API, with modifications, additions, or subtractions.
+**Derivative**: Software which includes at least some portion of the Mod other than the API.
 
-**Addon**: Software that extends on the Mod, but is not a derivative.
+**Addon**: Software that uses the API to interact with the Mod, but is not a derivative.
```

Comment:
I thought the dictionary definition worked quite well:
"cause to cover a larger area; make longer or wider."

Extends/expands/expounds, they all have similar meanings. All I'm trying to do here is make it clear that it's anything that does stuff with the mod on the code level (eg, a pipe that connects to my machine isn't an addon), but doesn't derive from it.